### PR TITLE
tests/*: rename bb tests

### PR DESCRIPTION
**Pull request by ajeddeloh**
Use dots and avoid non-alphanumeric characters. Try to group tests.

I am very open to new names for any of these if anyone has better ones.

### CODE REVIEW

File: tests/negative/regression/filesystem.go [hunk 1]
```diff
@@ -25,8 +25,7 @@ func init() {
 
 func VFATIgnoresWipeFilesystem() types.Test {
 	// Originally found in https://github.com/coreos/bugs/issues/2055
-
-	name := "Regression: VFAT ignores WipeFilesystem"
+	name := "regression.fs.ignoreswipe"
```
Comment (arithx) [inline tests/negative/regression/filesystem.go]: This PR seems to switch between `fs` & `filesystem` in test names. Is your intention that it's spelled out only if it's in the top level?

File: tests/negative/filesystems/symlinks.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ func init() {
 }
 
 func UsesSymlinks() types.Test {
-	name := "No Device"
+	name := "filesystems.pathusessymlinks"
```
Comment (arithx) [inline tests/negative/filesystems/symlinks.go]: Maybe `filesystem` to match the positive test's top level?

File: tests/negative/files/invalid_hash.go [hunk 3]
```diff
@@ -25,7 +25,7 @@ func init() {
 }
 
 func InvalidHash() types.Test {
-	name := "Invalid File Hash"
+	name := "file.verification.badhash.dataurl"
```
Comment (arithx) [inline tests/negative/files/invalid_hash.go]: Maybe `files.` to match other top-levels?

File: tests/positive/timeouts/timeouts.go [hunk 4]
```diff
@@ -99,7 +99,7 @@ func IncreaseHTTPResponseHeadersTimeout() types.Test {
 }
 
 func ConfirmHTTPBackoffWorks() types.Test {
-	name := "Confirm HTTP Backoff Works"
+	name := "timesouts.http.backoff"
```
Comment (arithx) [inline tests/positive/timeouts/timeouts.go]: Extra `s` in `timeouts`

**Comment by ajeddeloh** [comment]
fixed comments and tests

**Pull request merged**